docs/source/tutorials/design_flow/generate_fabric.rst

```diff
@@ -35,7 +35,7 @@ After finalizing your configuration file, you can run the task by calling the py
 
 .. code-block:: shell
 
-  python3 openfpga_flow/scripts/run_fpga_task.py basic_tests/generate_fabric 
+  python3 openfpga_flow/scripts/run_fpga_task.py openfpga_flow/tasks/basic_tests/generate_fabric 
```

Comment:
Actually both should work. 
- By default, the python script will search the task name under ``openfpga_flow/tasks/`` directory. 
- If not found, the python script will try to find the task under the exact path given by user.

Let me know if you see anything different?

Reply:
Actually you are right, both works, I should have checked first and then PR. Thank you.